app/models/user.rb

```diff
@@ -19,20 +27,26 @@ class User < ApplicationRecord
   has_many :subscriptions, through: :customers
 
   validates :email, presence: true
-  validates :password, presence: true
+  validates :password, presence: true, on: :create
```

Comment:
We don't want to allow empty passwords on update I think?

Reply:
this will not override password if its nil? (since the password digest should stay)

Reply:
how can this validation fail on touch?

Reply:
I updated this with 

```
validates :password, presence: true, if: lambda { new_record? || !password.nil? }
```

so we check that the password is not nil and not empty if its a new record and if its an update, we do not validate if password is nil (so will not be updated)